recipes-extended/xen/xen-source.inc

```diff
@@ -0,0 +1,5 @@
+LIC_FILES_CHKSUM = "file://COPYING;md5=d1a1e216f80b6d8da95fec897d0dbec9"
+
+XEN_URL = "git://github.com/xen-troops/xen.git;protocol=https;branch=xen-4.19-xt0.1"
+XEN_REL = "4.19"
+#XEN_REV = "${AUTOREV}"
```

Comment:
please no comments in the code. You should place XEN_REV here or, if you want to set XEN_REV from the upper layer - set XEN_REV ?= "${AUTOREV}"
You have set xen branch here have you? so please do all changes in one place

Reply:
Ok